.gitignore

```diff
@@ -29,6 +30,11 @@ go.work.sum
 # Shannon E2E config
 .shannon.config.yaml
 
+# Env
```

Comment:
Added these lines because the envoy branches use a `.env` file in the `auth_server` package so ensuring my local envs don't get committed.